masking_package/background_functions.py

```diff
@@ -0,0 +1,149 @@
+'''Background functions, functions the user does not have to know about'''
+
+def reverse_to_default(user_input, supposed_level, default_value):
+	'''Tests whether the user input is of the desired value level (e.g., bool).
+	If it is not, the default value of the package is used, instead.'''
```

Comment:
I think good documentation style (as far as I have seen it) is like this:
```suggestion
	'''
	Tests whether the user input is of the desired value level (e.g., bool).
	If it is not, the default value of the package is used, instead.
	'''
```